src/components/issues/ActionForm.tsx

```diff
@@ -106,74 +121,125 @@ const ActionForm: FC<ActionFormProps> = ({ taskId }) => {
     });
 
     return (
-        <form>
-            {isLoading && <Loader />}
-            <button
-                className={styles.card__top__button}
-                type="submit"
-                disabled={isAssigned}
-                onClick={handleSubmit}
-            >
-                Assign Task
-            </button>
```

Comment:
Just moved this button and loader to the bottom and added a few styles. Rest everything remains the same.

Reply:
Also added a started date input field.